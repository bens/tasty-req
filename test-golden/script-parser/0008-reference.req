1: POST /
---
{:r 1!}
