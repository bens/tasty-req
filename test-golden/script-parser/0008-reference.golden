[ Command
    { command'id = 1
    , command'always = False
    , command'method = "POST"
    , command'url = [ Right "/" ]
    , command'request_body = Nothing
    , command'response_body =
        Just (JsonCustom "r" (RespRef (Ref 1 Response [] (fromList []))))
    }
]
